components/brave_rewards/resources/extension/brave_rewards/_locales/en_US/messages.json

```diff
@@ -420,5 +420,23 @@
   "twitterTipsIconLabel": {
     "message": "Tip",
     "description": "Icon label for Twitter tips"
+  },
+  "verifiedPublisherNotification": {
+    "message": "Creator $name$ recently verified",
+    "description": "Notification text that tells user which publisher did just verified",
```

Comment:
Typo: Can delete `did` from this description, I think.

Reply:
done